frontend/src/components/BoardsList/BoardsList.jsx

```diff
@@ -0,0 +1,66 @@
+import { useQuery } from 'react-query';
+import Board from 'components/Board/Board';
+import { BoardWrapper } from './BoardList.styled';
+import { useState } from 'react';
+
+const BoardsList = () => {
+  const [userCords, setUserCords] = useState({ isSet: false });
+
+  const {
+    data: boardsData,
+    isLoading,
+    isError,
+  } = useQuery('boards', async () => {
+    return await fetch('/boards').then((response) => response.json());
+  });
+
+  if (isLoading || isError) {
+    return <></>;
+  }
+
+  navigator.geolocation.getCurrentPosition(
+    ({ coords }) => {
+      console.log('cords', coords);
+      setUserCords({ latitude: coords.latitude, longitude: coords.longitude, isSet: true });
+    },
+    () => setUserCords({ isSet: false }),
```

Comment:
Co robi ta linijka i po co nam ona?

Reply:
pierwszy argument funkcji, to co ma się stać normalnie, a drugi (czyli lionia 26) to co ma się wydarzyć w razie erroru